app/models/customer.py

```diff
@@ -0,0 +1,23 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    postal_code = db.Column(db.Integer)
+    phone = db.Column(db.String)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+    rentals = db.relationship('Rental', back_populates='customer', lazy=True)
```

Comment:
https://docs.sqlalchemy.org/en/14/orm/relationship_api.html
backref and back_populates do almost the same thing, but back_populates requires the property be explicitly configured on the other model, where backref does the configuration automatically. 

```suggestion
    rentals = db.relationship('Rental', backref='customer', lazy=True)
```